packages/tizen_log/lib/tizen_log.dart

```diff
@@ -0,0 +1,160 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+import 'package:ffi/ffi.dart';
+import 'package:flutter/foundation.dart';
+
+typedef _DLogNative = Void Function(
+    Int32, Pointer<Utf8>, Pointer<Utf8>, Pointer<Utf8>);
+typedef _DLog = void Function(int, Pointer<Utf8>, Pointer<Utf8>, Pointer<Utf8>);
+
+/// Provides the ability to use Tizen dlog logging service.
+/// More info: https://docs.tizen.org/application/native/guides/error/system-logs/
```

Comment:
The linked page only explains four log levels; debug, info, warn, and error. It's also a native API guide document where most content is irrelevant to Flutter developers. We should provide some guidance of each log level in the public doc comment for better developer experience. I have the following distinctions in mind but please share other opinions @flutter-tizen/maintainers:
- verbose: log messages of very detailed information for development.
- debug: log messages for development
- info: log messages for administration, typically used to report progress of the application.
- warn: to indicate problems that the program can tolerate but should be resolved whenever possible.
- error: to indicate problems that disturbs the normal workflow of the application, such as functional or performance limitations.
- fatal: to indicate problems that entirely blocks the normal workflow of the application.

The difference between verbose and debug logs seems to be in the "level of detail", since debug logs are always shown together with verbose logs but verbose logs can be filtered when priority filter is set to "debug".

references:
- https://developer.tizen.org/dev-guide/training/native-app/en/wearable/lesson_13/index.html
- https://stackoverflow.com/questions/25457858/android-log-levels

Reply:
I added description of each priority in the README and as comments for functions. I also replaced the link. What do you think?

Reply:
The log levels are already quite straightforward for most of developers to understand. I don't think we need to state the info in README.

Reply:
@m-bialek Thanks for the fix, I've added some following suggestions below. I don't mind removing it from README as long as it's stated in the doc comments.

Reply:
Also, providing a link to the official [native API documentation](https://docs.tizen.org/application/native/guides/error/system-logs) just makes sense because this plugin is a wrapper around the native API.

Reply:
Isn't that implementation detail? I agree if the link is not written in doc comments. Users just need to know how to use the API.

Reply:
Doc comments can provide any information regarding the API including its implementation details. Also the document is not just about the native API but provides general information about the dlog logging service and the dlogutil utility. Of course it would be better if there's an official dedicated document (end-to-end guide) for Dart users.

Reply:
> Doc comments can provide any information regarding the API including its implementation details.

It can if it's necessary but I didn't find any reasons why users need or want to read about native API but I agree that dlog logging service and dlogutil utility are required information. 

If we can't avoid linking the document, at least it should say something like:
```
/// [Log] is a wrapper of Tizen's native API, more info: 
/// https://docs.tizen.org/application/native/guides/error/system-logs/.
```

However if this plugin is only intended for inside developers who already know about how Tizen APIs are structured then I'm good with the original comment.

Reply:
A "plugin" (by its definition) implicitly indicates that it's wrapping some underlying native API. A user has no idea about what dlog is and what it's intended for may want to learn what they can do with the API, by reading any related documentation (the native API guide in this case) if there's no Dart counterpart of it. Can you even guess where you can see the dlog output if there's only a few lines of doc comments without the link?

Reply:
My intention was to avoid the link if it's possible or guide why the link is there. Reading about native API(or any C code examples) is still unnecessary even if the user doesn't know about dlog. The reason why the link should be in the comment in this case is because of dlog logging service and dlogutil utility, so I agree adding the link but suggested that we write more than "More info: \<link\>" when we're providing much more information than the user want.